cloud_governance/policy/aws/cost_billing_reports.py

```diff
@@ -40,7 +40,14 @@ def get_date_ranges(self, days: int = 0):
         """
         end_date = datetime.datetime.now()+datetime.timedelta(1)
         if days == 0:
-            start_date = f"{end_date.year}-{end_date.month-1}-01"
+            month = end_date.month
```

Comment:
Very risky when handling it like this, what happened in end of month ?

Reply:
today month is 0, when I decrease -1 it becomes 0